src/tests/app.spec.js

```diff
@@ -6,3 +6,4 @@ const request = require('supertest').agent(server.listen());
 chai.should();
 
 require('../resources/account/account.test')(request);
+require('../resources/user/user.test')(request);
```

Comment:
At maqpie we do not have such common file, that groups all tests. Here is an example: https://github.com/maqpie/maqpie/blob/master/chat-api/src/resources/widget/app/app.controller.spec.js. 

What if we put first four rows into `app.test.js` or something like this and would just call it from `account.test`, `user.test`, etc? 

We also could move based test builder over to the resources folder and completely remove tests folder :) and keep tests as close as possible to the files with business logic.

P.S. Here is how tests being executed: https://github.com/maqpie/maqpie/blob/master/chat-api/package.json#L31